matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/DefaultCryptoService.kt

```diff
@@ -1301,6 +1308,31 @@ internal class DefaultCryptoService @Inject constructor(
         cryptoStore.logDbUsageInfo()
     }
 
+    override fun ensureOutboundSession(roomId: String) {
+        cryptoCoroutineScope.launch(coroutineDispatchers.crypto) {
+            // Ensure to load all room members
+            loadRoomMembersTask.execute(LoadRoomMembersTask.Params(roomId))
```

Comment:
Always catch execute calls.
not sure what the strategy would be on fail though :/ send any way

Reply:
Done.